Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -634,6 +638,10 @@ def SwapKeepAndDrop(l):
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_MergedTrackTruth_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_StripDigiSimLink_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_PixelDigiSimLink_*')
+approxSiStripClusters.toModify(FEVTDEBUGHLTEventContent,
+                              outputCommands = FEVTDEBUGHLTEventContent.outputCommands+[
+                                  'keep *_hltSiStripClusters2ApproxClusters_*_*'
```

Comment:
This seems unnecessary, because `FEVTDEBUGHLTEventContent` is a superset of `FEVTDEBUGEventContent`, and the latter already includes `hltSiStripClusters2ApproxClusters` due to L625.

Reply:
I just retested and it seems to be necessary as there is a drop `*_hlt*_*_*` command hidden somewhere in the appends.